pkgs/applications/misc/termite/default.nix

```diff
@@ -1,20 +1,25 @@
-{ stdenv, fetchFromGitHub, pkgconfig, vte-ng, gtk3, ncurses, wrapGAppsHook }:
+{ stdenv, fetchFromGitHub, fetchpatch, pkgconfig, vte-ng, gtk3, ncurses, wrapGAppsHook }:
 
 stdenv.mkDerivation rec {
   name = "termite-${version}";
-  version = "14";
+  version = "15";
 
   src = fetchFromGitHub {
     owner = "thestinger";
     repo = "termite";
     rev = "v${version}";
-    sha256 = "0dmz9rpc2fdvcwhcmjnhb48ixn403gxpq03g334d1hgjw2hsyx7x";
+    sha256 = "0hp1x6lj098m3jgna274wv5dv60lnzg22297di68g4hw9djjyd2k";
     fetchSubmodules = true;
   };
 
   # https://github.com/thestinger/termite/pull/516
   patches = [ ./url_regexp_trailing.patch ./add_errno_header.patch
-              ] ++ stdenv.lib.optional stdenv.isDarwin ./remove_ldflags_macos.patch;
+    # Fix off-by-one in select_text() on libvte >= 0.55.0
```

Comment:
Do you know when this patch could go away? If so, please add a note.

Reply:
It's from upstream's tree so I would expect it to be included in the next release.  Will add a note, thanks!